.github/renovate.json5

```diff
@@ -181,6 +182,15 @@
         "v1.11"
       ]
     },
+    {
+      "matchPackageNames": [
+        "docker.io/library/busybox"
+      ],
+      "allowedVersions": "<1.35",
```

Comment:
Should this be at least `<=1.35`? My understanding is that as-is Renovate will never update the image.

Reply:
fixed